packages/backend/README.md

```diff
@@ -41,29 +41,48 @@ Optionally if you want to speed up the price collection obtain an api key from:
 Once you have everything create a `.env` file that configures the app's
 environment variables. One variable per line `KEY=value`.
 
-- `ETHERSCAN_API_KEY` - Api key for Etherscan
-- `ALCHEMY_API_KEY` - Api key for Alchemy
-- `DATABASE_URL` - Database url used in production deployment
+#### Required
```

Comment:
Maybe this section should be split between required locally and required on production?